bootstrap_resume_sass/scss/main.scss

```diff
@@ -0,0 +1,231 @@
+$primary-color: green;
+$font-size: 1.5vw;
+$third: 33.333%;
+$ten-pixel: 10px;
```

Comment:
I am not sure about using hardcoded name :) You can for example use "large-spacing, medium-spacing etc"